### DISCUSSION TITLE
What is powering Filament's Community (Forum) ?

### DISCUSSION THREAD

**Discussion by vesper8** [filamentphp/filament#15138] [General]
I'm talking about this page https://filamentphp.com/content

This forum-like community discussion section. Is this powered by Filament itself or is another library being used?

I would like to set up something similar for a project.

Many thanks!

**Comment by xibn** [comment]
> I'm talking about this page https://filamentphp.com/content
> 
> This forum-like community discussion section. Is this powered by Filament itself or is another library being used?
> 
> I would like to set up something similar for a project.
> 
> Many thanks!

Check out this repository, it should be the complete website afaik: https://github.com/filamentphp/filamentphp.com 👍
